README.md

```diff
@@ -1,15 +1,60 @@
-[![JetBrains obsolete project](https://jb.gg/badges/obsolete.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
+# kotlinx-io
 
+[![Kotlin Alpha](https://kotl.in/badges/alpha.svg)](https://kotlinlang.org/docs/components-stability.html)
+[![JetBrains incubator project](https://jb.gg/badges/incubator.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
+[![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
+[![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlinx/kotlinx-io/0.1.16)](https://central.sonatype.com/artifact/org.jetbrains.kotlinx/kotlinx-io/0.1.16)
```

Comment:
Probably should be `0.2.0-alpha-dev1`

Reply:
There's no such version at the moment. I was thinking of updating the version in the badge after publishing artifacts.

Reply:
Probably worth omitting the badge until then.

Up to you, no strong opinions here